_shared/project/bin/logger

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
```

Comment:
This is again just the exact same `logger` script that used to live at `pyramid-app/{{ cookiecutter.slug }}/bin/logger`, just moved into `_shared` and git doesn't recognise it as a file move because they're now a symlink at the original file location.